### ISSUE TITLE
JER SF 

### ISSUE THREAD

**Issue by jpata** [scikit-hep/coffea#191]
**Describe the bug**
Latest CMS JER SF files bin the scale factor in Pt and Eta, but bin indices are not produced in the correct 1d format [here](https://github.com/CoffeaTeam/coffea/blob/master/coffea/lookup_tools/jersf_lookup.py#L90). One option could be that `jersf_lookup._parms` should be a jagged array, instead of a flat array.

**To Reproduce**
Can be reproduced using the unit test here: https://github.com/CoffeaTeam/coffea/pull/190

**Expected behavior**
The 2d lookup should succeed for the latest JER SF file.

**Output**
```python
Traceback (most recent call last):
  File "tests/test_jetmet_tools.py", line 236, in <module>
    test_jet_resolution_sf_2d()
  File "tests/test_jetmet_tools.py", line 128, in test_jet_resolution_sf_2d
    resosfs = resosf.getScaleFactor(JetPt=test_pt, JetEta=test_eta)
  File "/storage/user/jpata/hmm/dev/hepaccelerate-cms/coffea/coffea/jetmet_tools/JetResolutionScaleFactor.py", line 143, in getScaleFactor
    sfs.append(func(*tuple(args)))
  File "/storage/user/jpata/hmm/dev/hepaccelerate-cms/coffea/coffea/lookup_tools/lookup_base.py", line 29, in __call__
    retval = self._evaluate(*tuple(inputs))
  File "/storage/user/jpata/hmm/dev/hepaccelerate-cms/coffea/coffea/lookup_tools/jersf_lookup.py", line 90, in _evaluate
    parm_values = self._parms[bin_tuple]
IndexError: index 3 is out of bounds for axis 1 with size 3
```

**Comment by lgray** [comment]
OK - it shouldn't be too bad to add jagged binning in for this.

Thanks for finding it!

**Comment by jpata** [comment]
I'll see if I can introduce jagged arrays to the JER based on the existing JEC code!

**Comment by lgray** [comment]
Fixed in #190
